.changeset/wet-eyes-clean.md

```diff
@@ -0,0 +1,9 @@
+---
+"@keystone-next/example-ecommerce": patch
+"@keystone-next/keystone": major
+"@keystone-next/types": major
+---
+
+Updated onConnect arguments to provide `createContext` and `knex` / `mongoose` / `prisma` (depending on the adapter in use) instead of the BaseKeystone instance
```

Comment:
```suggestion
Changed the `config.db.onConnect` argument to accept a `KeystoneContext` instance, created with `{ skipAccessControl: true }`, rather than a `BaseKeystone` instance.
```